### PR TITLE
PR for wishlist feature

### PR DESCRIPTION
**Pull request by narharikale**
- add to wishlist
- remove from wishlist
- wishlist count
- only logged in users can able to add to wishlist

### CODE REVIEW

File: src/pages/auth/login.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ function Login() {
                             {isPasswordVisible ? 'visibility' : 'visibility_off'}
                         </span>
                     </div>
-                    { isErr ? <small className="color-red-600 bg-red-200 p-alert w-100-per">&#9888; Some error occurred </small> :<></> }
+                    { isErr ? <small className="color-red-600 bg-red-200 p-alert w-100-per">&#9888; Some error occurred </small> :null }
```
Comment (vijaytembugade) [inline src/pages/auth/login.jsx:46]: use `&&` for booleans !

File: src/context/wishlist-context.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                    console.error(error, "wishlist useEffect")
+                }
+            })()
+        }
+        else {
+            setWishlistData([])
+        }   
+        
+        
+    }, [isAuth]);
```
Comment (vijaytembugade) [inline src/context/wishlist-context.js:67]: Suggestion: You can create reducer, although its looks good like this too

**Review by vijaytembugade — APPROVED** [review]
Good work Narhari!
Just create separate reducers for context.
Using barrel export is good practise, that you've used.

File: src/context/wishlist-context.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                });
+                setWishlistData(res.data.wishlist)
+            } catch (error) {
+                console.error(error)
+            } 
+        }
+        else {
+            navigate('/login')
+        }
+    }
```
Comment (MiheerTamkhane) [inline src/context/wishlist-context.js:28]: suggestion : Make service folder for this kind of functions.

**Review by MiheerTamkhane — APPROVED** [review]
Code looks good and readable, Nice work man Good to go.

**Pull request merged**